pkg/apis/logging/v1/kibana_types.go

```diff
@@ -32,8 +32,6 @@ type KibanaStatus struct {
 	Conditions  map[string]ClusterConditions `json:"clusterCondition,omitempty"`
 }
 
-type ClusterConditions []ClusterCondition
```

Comment:
why are we removing this here but adding it in the elasticsearch_types?

Reply:
Type aliases should live close to their referenced/aliased types. IMHO it enhances readability.

Reply:
so this wasn't being used here nor in elasticsearch_types before? why not just drop it here and leave it as that?

Reply:
It was used in both types but `ClusterCondition` lived in `elasticsearch_types.go` while this alias in `kibana_types.go`